### PR TITLE
Devcenter: passing files/dependencies from one build to another

### PR DESCRIPTION
**Pull request by BanyikAnna**

### CODE REVIEW

File: docs/caching/caching-build-build.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+
+You can pass artifacts from one build to another with the [`Bitrise.io Cache:Pull`](https://www.bitrise.io/integrations/steps/cache-pull) and [`Bitrise.io Cache:Push`](https://www.bitrise.io/integrations/steps/cache-push) steps in your workflow. Insert the `Bitrise.io Cache:Pull` step right after the `Git Clone Repository` step and the `Bitrise.io Cache:Push` step at the end of your workflow. Note, that in the first build there is no cache archive yet that can be downloaded with the `Bitrise.io Cache:Pull` step. Once the first build has successfully run on Bitrise, the cache archive files will be uploaded to the cloud storage with the `Bitrise.io Cache:Push` step. The next build will be able to access the files stored on the cloud with the `Bitrise.io Cache:Pull` step.
+
+!!! important
+    You can create a cache archive of the files that were created during the building of your master branch or of the a new build that was NOT started by a pull request.
```
Comment (godrei) [inline docs/caching/caching-build-build.md]: or __of the a__ new build -> is it grammatically correct?

**Comment by godrei** [comment]
For me, this article seems to be a functional description of our cache system. Based on the title I thought it will be about this: https://blog.bitrise.io/pass-artifacts-between-builds

File: docs/caching/caching-build-build.md (hunk 1 above)
Comment (BanyikAnna) [reply docs/caching/caching-build-build.md]: nope, no need for "a" . Thank you!

**Comment by BanyikAnna** [comment]
not needed in this form, will do another PR to modify About cache section and add the info there.

**Pull request closed without merging**